tests/model/test_model.py

```diff
@@ -434,6 +434,33 @@ def test_react_to_message_for_not_thumbs_up(self, model):
         with pytest.raises(AssertionError):
             model.react_to_message(dict(), 'x')
 
+    @pytest.mark.parametrize('recipient_user_ids, status', [
+        ([5140], 'start'),
+        ([5179], 'stop')
+    ])
```

Comment:
I've added variations to the number of recipients in the list, and have used nested parameterize statements, so as to have good coverage since we have a reasonably small number of variants.